presto-product-tests/src/main/java/com/facebook/presto/tests/utils/JdbcDriverUtils.java

```diff
@@ -13,7 +13,12 @@
  */
 package com.facebook.presto.tests.utils;
 
+import com.facebook.presto.jdbc.PrestoConnection;
+
 import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
```

Comment:
It's possible this addition ended up in the wrong commit to to re-ordering during the squash process, but the build would have failed if they were not used in the final version of the code which is made up of all commits combined.